tests/testthat/test-oauth.R

```diff
@@ -53,6 +53,30 @@ test_that("oauth_encode1 works", {
   expect_equal(orig_string, restored_string)
 })
 
+test_that("custom oob callback values can be provided", {
```

Comment:
TBH I don't think this is worth it; in httr2 we'll refactor to make this sort of testing easier, but for now I think it's better to omit as the test will be fragile.

Reply:
Removed.